support-frontend/assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -145,6 +145,8 @@ export type CountryMetaData = {
   headerClasses?: string,
   // URL to fetch ticker data from. null/undefined implies no ticker
   tickerJsonUrl?: string,
+  // Optional message to display at the top of the form
+  formMessage?: React$Element<string>,
```

Comment:
does this come from a file in s3 or something?

Reply:
Good question. The `CountryMetaData` type is actually used in this file - I've just pushed a commit to clarify where it's used

Reply:
So for the AU campaign the `formMessage` will be set in this file